### PR TITLE
RI-7547 Stop showing tooltip suggestions in the editor when inserting saved queries

### PR DESCRIPTION
**Pull request by valkirilov**
# Description

Disable the "suggestions tooltip" in the code editor on the **Vector Search** page when inserting a saved query, because it was hiding the **Run** button (_on a specific screen size_).

| Before | After |
| - | - |
image|image

# How it was tested

1. Open the **Databases** page and connect to an existing database (or establish a connection with a new instance)
2. Go to the **Search** tab from the main navbar

_Note: Make sure to create a new index via the preset sample datasets, following the Vector Search onboarding wizard._

3. Open the **Saved Queries** panel from the top right corner
4. Click on the "**Insert**" button of any of the provided queries

At this point, the query should be added automatically to the code editor, but the "Run" button should remain visible.

https://github.com/user-attachments/assets/ee63c13e-7078-4f32-b518-0940d6ecf3e4

### CODE REVIEW

**Review by ArtemHoruzhenko — APPROVED** [review]
lgtm

File: redisinsight/ui/src/pages/workbench/components/query/Query/Query.tsx [hunk 1]
```diff
@@ -493,6 +493,12 @@ const Query = (props: Props) => {
     const { data, forceHide, forceShow } = getSuggestions(editor, command)
     suggestionsRef.current = data
 
+    // Prevent suggestions if editor is not focused or cursor is not set
+    if (!editor.hasTextFocus() || !editor.getPosition()) {
```
Comment (KrumTy) [inline redisinsight/ui/src/pages/workbench/components/query/Query/Query.tsx:497]: do we want test coverage for handleSuggestions?

Comment (valkirilov) [reply redisinsight/ui/src/pages/workbench/components/query/Query/Query.tsx:497]: There are ~0 tests related to this functionality, but I can try to build up some test coverage.

Comment (valkirilov) [reply redisinsight/ui/src/pages/workbench/components/query/Query/Query.tsx:497]: Bad news, sir. It seems like there was a reason for not having any tests related to the code editor. In the unit tests, currently we have mocked the Monaco editor, so it's not rendered at all. Nevertheless, I tried to see if there is a way to use the current mock (or extend it), but for the purpose of this use case, we rely on internal events (for the Monaco editor) to be in place and trigger the suggestion popup. 

So, I was not able to create any tests to verify this behavior, via unit tests. 

Still, there is a way to cover this with an e2e test with Playwright, since we have the full user experience there, but this is kind of an edge case, so I'm not sure whether we need it. What do you think? Should I invest time to implement such an e2e test case?

Comment (valkirilov) [reply redisinsight/ui/src/pages/workbench/components/query/Query/Query.tsx:497]: Ok, I have added two simple checks to the existing e2e tests for the new Search page, so now we verify whether the suggestions tooltip is visible in the code editor:
- it should be visible when you manually type in commands
- it shouldn't be visible when you insert commands from the "Saved Queries"

**Review by pd-redis — APPROVED** [review]
